paket.dependencies

```diff
@@ -17,9 +17,10 @@ nuget Feliz prerelease
 nuget Feliz.Bulma
 nuget Feliz.Bulma.Tooltip
 nuget Fable.Browser.Css
-nuget Fable.Browser.Dom
-nuget Fable.Browser.Event
+nuget Fable.Browser.Dom == 2.4.4
+nuget Fable.Browser.Event == 1.4.5
 nuget Fable.Browser.MediaQueryList
+nuget Fable.Browser.Navigator == 2.0.0
```

Comment:
Is there conflict between dependencies for you to need to lock the versions ?

Reply:
It pulls to latest version and something broke. It some versions mismatch, and i feel that REPL is a bit picky. Also it produce larger diff in lock file. If you don't mind updating deps, I'm fine :) also I can work separately on that.

Reply:
No problem, keep it like that I will have a look when testing your PR.